### PR TITLE
Panda & Tiger Level complete

### PR DESCRIPTION
**Pull request by ralphos**
Got most of the homework done, but due to lack of time, my solution to the Tiger level was pretty crappy. I had some real difficulty conceptualizing how I would approach the two-step workflow, i.e. asking the user two questions and then coming to a conclusion. I ended up with some very basic control flow. 

Q: I created the seed file for all the page creates. In there I also set a 'page' variable for the parent page. The only way I could get it to work is if I also set the 'page' variable in the adventure.rb file (see `page = Page.all.first`) and I'm not sure why?

PS wrote my first test BEFORE writing the code. It was only one test and it was pretty standard but got my first taste of TDD, so I guess that's a milestone :)

### CODE REVIEW

**Comment by ralphos** [comment]
I'll comeback and revisit the Eagle level if I have time - looks like it requires an association (has_many/belongs_to) with foreign keys/ids stored instead of parent id's?

File: adventure.rb [hunk 1]
```diff
@@ -4,11 +4,9 @@
 require_relative 'db/setup'
 require_relative 'models/page'
 require_relative 'models/book'
+require_relative 'db/seed' # lesson learned: this has to be placed below the page/book models. 
 
-page = Page.create(starting_point: true, content: "You wake up on a road. It's foggy and dampy. In your bag is 30 gold pieces and a bacon sandwich. Which do you choose?")
-Page.create(conclusion: true, parent_id: page.id, content: "Go into the forest")
-Page.create(conclusion: true, parent_id: page.id, content: "Walk down the road")
-
+page = Page.all.first # shouldn't I be able to leave this out and let the page variable be set in the seed file?
```
Comment (jwo) [inline adventure.rb:9]: Even if you could get that to work, say with a `@page` ... I think it'd be hard to follow where exactly `@page` came from, so I wouldn't

**Comment by jwo** [comment]
Getting your taste of TDD is DEFINITELY an accomplishment.

I think this works fine, but you probably don't need answer_one and answer_two methods... You could create an Answer class that behaves differently based on its page -- extract those decisions out as far as you can.

Regarding the association -- Sure, that'l work, or you can literally just store the IDs in a string column as a comma separated list. Don't need to be crazy formal if you're not dealing with data integrity .

File: adventure.rb (hunk 1 above)
Comment (ralphos) [inline adventure.rb:9]: Got it.

**Comment by ralphos** [comment]
OK cool. I'll see if I have some time to revisit this but I'm quite busy and I want to catch up with this weeks exercises so maybe next week I can go over them again.
